examples/intel-optimized-mysql-server_vpc_creation_testing/main.tf

```diff
@@ -4,29 +4,37 @@
 
 # Provision Intel Optimized AWS MySQL server
 module "vpc" {
-  source = "github.com/intel/terraform-intel-aws-mysql"
-  name = "my-vpc"
-  cidr = "10.0.0.0/16"
+  source = "terraform-aws-modules/vpc/aws"
```

Comment:
I would add a version constraint in here. It could be relatively forgiving where it's `~> 3.18.1` so that you are just allowing minor version changes

Reply:
Thanks! Done.